src/Functions/minus.cpp

```diff
@@ -13,7 +14,7 @@ struct MinusImpl
     static const constexpr bool allow_string_integer = false;
 
     template <typename Result = ResultType>
-    static NO_SANITIZE_UNDEFINED Result apply(A a, B b)
+    static NO_SANITIZE_UNDEFINED Result apply(A a, B b, NullMap::value_type * m [[maybe_unused]] = nullptr)
```

Comment:
It's better not to change every `apply()` function.

I mean `FunctionBinaryArithmetic::executeNumeric()` calls `BinaryOperationImpl<A, B, Op>::process()`, and usually `BinaryOperationImpl<A, B, Op>` is the same as `BinaryOperation<A, B, Op>`, but it doesn't have to be always the same.

For example look at the definition of `BinaryOperationImpl<UInt64, UInt8, ModuloImpl<UInt64, UInt8>>` - it provides its own `process()` function.

So you can make your own `process()` function for your functions `moduloOrNull` and `divideOrNull` without changing functions `apply()` everywhere.

Reply:
I  first idea was that functions could produce NULL value, like divide zero, providing a way to do this if later other functions needs this feature.